### ISSUE TITLE
Edit Properties Problem

### ISSUE THREAD

**Issue by GrobyKen** [TheGameCreators/GameGuruRepo#912]
Latest version. Edit Properties on an object. Works fine first time but blacks out object if you return to adjust properties again.

**Comment by AmenMoses** [comment]
Max or Classic?

**Comment by GrobyKen** [comment]
Yes sorry mate should have said. Max. Also anyone have an idea when Max will 'go live' and is there any point at the moment of uploading stuff to the store for Max?

**Comment by synchromesh62** [comment]
Depends what you creating. 
There is an option to say Max compatible so basically anything you create for GG and works in Max can be tagged for both.

**Comment by rvanner** [comment]
Object properties is changing this week so please check the next version and report back to this thread thanks.

**Comment by rvanner** [comment]
@synchromesh62  Is this fixed now for you?

**Comment by GrobyKen** [comment]
Will download new version and check tomorrow. Then post results.
